doc/source/working_with_kiwi/xml_description.rst

```diff
@@ -780,7 +780,27 @@ removed), whose name is specified via the mandatory `name` attribute:
    </image>
 
 which adds the package `udev` to the list of packages to be added to the
-initial filesystem.
+initial filesystem. Note, that the value that you pass via the `name`
+attribute is passed directly to the used package manager. Thus, if the
+package manager supports other means how packages can be specified, you may
+pass them here too. For example rpm based package managers (like
```

Comment:
"Here", "there" etc. are usually a source of ambiguity. I guess, it refers to the package manager and not the `name` variable. If you can avoid it you could replace "here" with something else. One idea:

```
Thus, if the package manager supports other means how packages can be specified, you may
pass them in this context too.
```

Maybe there are better words. Or you could rephrase the sentence a bit. I'll leave it up to you. :wink: